lib/pluginmanager/xpack/pack.rb

```diff
@@ -0,0 +1,82 @@
+# encoding: utf-8
+require "pluginmanager/errors"
+
+module LogStash module PluginManager module Xpack
+  # A object that represent the directory structure
+  # related to the actual gems in the extracted package.
+  #
+  # Example of a valid structure, where `logstash-output-secret` is the actual
+  # plugin to be installed.
+  #.
+  # ├── dependencies
```

Comment:
It will be called the name of the pack, so for a monitoring pack you will get something like this.

logstash-monitoring.zip
 - logstash-monitoring/
      - dependencies/
         - mydependecyXXXX.gem
      - logstash-monitoringXXX.gem